<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Players Stat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="show_b">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>֍ Show your games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_b">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>← Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showPlayer_b">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>֍ Show Players</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAll_b">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>֍ Show ALL games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAllGames_b">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>֍ Delete ALL Games</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
